package.json

```diff
@@ -8,59 +8,59 @@
 		"test": "./tools/scripts/use-make-instead"
 	},
 	"devDependencies": {
-		"@babel/core": "7.19.3",
-		"@babel/preset-env": "7.19.4",
+		"@babel/core": "7.20.5",
+		"@babel/preset-env": "7.20.2",
 		"@babel/preset-react": "7.18.6",
 		"@babel/preset-typescript": "7.18.6",
-		"@changesets/cli": "2.25.0",
-		"@emotion/babel-plugin": "11.10.2",
+		"@changesets/cli": "2.25.2",
+		"@emotion/babel-plugin": "11.10.5",
 		"@emotion/react": "11.0.0",
 		"@guardian/browserslist-config": "workspace:*",
 		"@guardian/eslint-config": "workspace:*",
 		"@guardian/eslint-config-typescript": "workspace:*",
 		"@guardian/prettier": "workspace:*",
 		"@manypkg/get-packages": "1.1.3",
-		"@nrwl/cli": "15.0.13",
-		"@nrwl/devkit": "15.0.13",
-		"@nrwl/eslint-plugin-nx": "15.0.13",
-		"@nrwl/jest": "15.0.13",
+		"@nrwl/cli": "15.2.1",
+		"@nrwl/devkit": "15.2.1",
+		"@nrwl/eslint-plugin-nx": "15.2.1",
+		"@nrwl/jest": "15.2.1",
 		"@nrwl/nx-cloud": "latest",
-		"@nrwl/workspace": "15.0.13",
-		"@storybook/addon-a11y": "6.5.12",
-		"@storybook/addon-docs": "6.5.12",
-		"@storybook/addon-essentials": "6.5.12",
-		"@storybook/addon-links": "6.5.12",
+		"@nrwl/workspace": "15.2.1",
+		"@storybook/addon-a11y": "6.5.13",
+		"@storybook/addon-docs": "6.5.13",
+		"@storybook/addon-essentials": "6.5.13",
+		"@storybook/addon-links": "6.5.13",
 		"@storybook/addons": "^6.5.13",
-		"@storybook/builder-webpack5": "6.5.12",
-		"@storybook/manager-webpack5": "6.5.12",
-		"@storybook/react": "6.5.12",
+		"@storybook/builder-webpack5": "6.5.13",
+		"@storybook/manager-webpack5": "6.5.13",
+		"@storybook/react": "6.5.13",
 		"@storybook/storybook-deployer": "2.8.16",
 		"@storybook/theming": "^6.5.13",
 		"@swc-node/register": "1.5.4",
-		"@swc/core": "1.3.8",
-		"@types/babel__core": "7.1.19",
-		"@types/jest": "29.1.2",
+		"@swc/core": "1.3.20",
+		"@types/babel__core": "7.1.20",
+		"@types/jest": "29.2.3",
 		"@types/node": "18.11.9",
 		"@types/react": "17.0.1",
-		"@typescript-eslint/parser": "5.42.0",
-		"babel-loader": "8.2.5",
-		"eslint": "8.25.0",
-		"husky": "8.0.1",
-		"jest": "29.2.0",
-		"jest-environment-jsdom": "29.2.0",
-		"lint-staged": "13.0.3",
-		"nx": "15.0.13",
-		"prettier": "2.7.1",
+		"@typescript-eslint/parser": "5.44.0",
+		"babel-loader": "9.1.0",
+		"eslint": "8.28.0",
+		"husky": "8.0.2",
+		"jest": "29.3.1",
+		"jest-environment-jsdom": "29.3.1",
+		"lint-staged": "13.0.4",
+		"nx": "15.2.1",
+		"prettier": "2.8.0",
 		"react": "17.0.1",
 		"react-dom": "17.0.1",
 		"require-from-string": "2.0.2",
 		"sort-package-json": "2.1.0",
 		"ts-jest": "29.0.3",
 		"ts-node": "10.9.1",
 		"tsconfig-paths-webpack-plugin": "4.0.0",
-		"tslib": "2.4.0",
-		"type-fest": "3.1.0",
-		"typescript": "4.8.4",
+		"tslib": "2.4.1",
+		"type-fest": "3.3.0",
+		"typescript": "4.9.3",
```

Comment:
I'd be cautious of this getting too far ahead of the libs / packages

Reply:
I thought all of the packages (apart from `eslint-config-typescript` which is pinned to `typescript@4.2.2`) just used the workspace version. Confused about how this could be getting too far ahead, am I missing something?

Reply:
ah, really interesting problem for library maintainers, thanks for pointing it out! i'll put this back to where it was for now.

 it'd be good to have a wider discussion about how we decide when it's "allowed" to upgrade deps that could affect consumers of our libraries... it seems really unclear/ ill defined right now. Did some reading around it, an interesting issue: https://github.com/microsoft/TypeScript/issues/36207 and the project they mention that "downlevels" d.ts files.

Reply:
Maybe a good one to bring up at CSI meetings? I think its less a question of being allowed and more thinking in terms of balance between what upgrading deps gives us (security, bug fixes etc) vs the challenges it creates for the other platforms that consume these libs packages etc. 

If teams don't have time to upgrade TS on their platforms and as a result can't use Source we're shooting ourselves in the foot there so its always a compromise. Having everything all new and shiny means nothing if no one uses it :)